patient-care/webapp/client/manageObjects/manageObjects.js

```diff
@@ -124,6 +124,68 @@ Template.manageObjectsGrid.onCreated(function () {
   });
 });
 
+Template.manageObjectsGrid.onRendered(function () {
+  let instance = this;
+
+  // NOTE: a deferred versino of refreshSticky is passed down all
+  // the way to the create/showTemplate level
+
+  // keep track of whether it's been stickied yet
+  // start this out as false because it hasn't been stickied yet
+  let stickied = false;
+
+  // Make the manage object detail sticky so it moves down
+  // the page as the user scrolls down.
+  function refreshSticky() {
+    // reactively watch the count of the items on the left so it'll update
+    // when it changes, but don't do anything with the return value
+    let hi = getObjects(instance).count();
```

Comment:
I'm confident there's a better way to depend reactively on a variable, but I don't know enough meteor to tell you what that is.